_pages/showcase.md

```diff
@@ -0,0 +1,56 @@
+
+The full details of the financial aid program from last year are available here. As with previous years, speakers will be given priority for financial aid requests within reasonable limits. Financial assistance requests are evaluated separately and cannot be considered guaranteed even if your talk is accepted.
+
+Some speakers may have attended other conferences where full costs have been met, including travel and accommodation. As a not-for-profit, community-run conference, this is generally not the case at PyCon AU. 
+
+## Code of Conduct
+PyCon Australia 2019 has a Code of Conduct [https://2019.pycon-au.org/conduct/](https://2019.pycon-au.org/conduct/) which outlines expectations of behaviour and appropriate presentation content. The goal of the code of conduct is to facilitate a safe, welcoming and inclusive environment. Presenters are required to be familiar with and abide by the code of conduct. If you have any questions about the suitability of any aspect of your proposed talk or require any additional information, please email us at [program@pycon-au.org](mailto:program@pycon-au.org).
+
+## Presentation Length
+The length for talks is 10 minutes. 
+
+## Presentation Video Recording
+Every session at PyCon Australia will be professionally recorded and the video uploaded on our YouTube channel. If there are concerns with this please raise them and we will endeavour to work with you.
+
+## How to Submit
+Fill in this form: [http://cmp.ac/pyconau-2019-student-showcase](http://cmp.ac/pyconau-2019-student-showcase)
+
+## Proposal Selection Process
+Selecting the proposals to include in the conference is complex. We take the process very seriously, and respect and value the effort that every submitter has taken in preparing their proposal. This means that the selection process takes some time to get through, although we endeavour to be efficient and well-organised.
+
+We will let students know whether their talks have been accepted, not selected this year, or waitlisted by .
+It might seem obvious, but the main reason a talk might not be selected is simply because of the constraint on the overall number of talks we can include at the venue.
+
+## If you would like feedback on your proposal
+If you have not yet submitted a proposal but would like feedback on your ideas before submitting, get in touch by emailing events@aca.edu.au or program@pycon-au.org. Or, if you are feeling particularly brave, just tweet the question and mention @pyconau and @auscompacademy!
+
+Alternatively, if you want to go ahead and submit a new proposal, please put any questions you have in the form, and the reviewers will see this and respond.
+
+If you wish to ask for feedback on an existing proposal, then please email us directly at [program@pycon-au.org](mailto:program@pycon-au.org), as we may not be alerted to changes in the proposal and therefore may not realise in a timely manner that a response is required.
```

Comment:
Please change to speakers@2019.pycon-au.org

Reply:
Done